Libraries/Image/RCTImageLoader.mm

```diff
@@ -947,8 +947,8 @@ - (RCTImageLoaderCancellationBlock)decodeImageData:(NSData *)data
           // Decompress the image data (this may be CPU and memory intensive)
           UIImage *image = RCTDecodeImageWithData(data, size, scale, resizeMode);
 
-#if !TARGET_OS_OSX && RCT_DEV // TODO(macOS GH#774)
-          CGSize imagePixelSize = RCTSizeInPixels(image.size, UIImageGetScale(image)); // TODO(macOS GH#774)
+#if !TARGET_OS_OSX && RCT_DEV // [macOS]
```

Comment:
No, because originally that endif was for `#if RCT_DEV`, that we modified to also have `!TARGET_OS_OSX`

https://github.com/facebook/react-native/blob/8bef1b345b010b8566dd6e33a42e9cd72414b251/Libraries/Image/RCTImageLoader.mm#L981